### DISCUSSION TITLE
How is this course so far?

### DISCUSSION THREAD

**Discussion by unfor19** [unfor19/kubernetes-localdev#6] [General]
Hi everyone :wave:

For those of you who "took" this course (fully or partially), how is it so far? Is there any topic that needs clarification?

I'm super curious; any feedback is appreciated.

**Comment by ranzanici** [comment]
it's very interesting, still going through it slowly to understand all steps

**Reply by unfor19** [reply]
Thank you! Let me know if you think I should shed more light on some areas

**Comment by lustoerk** [comment]
I like it very much, thanks alot for your work! I have just two thoughts:
- why does the host need https access to minikube?
- I put OAUTH2_PROXY_CLIENT_ID="google_oauth2_{kubemaster}_{myID}" instead of  OAUTH2_PROXY_CLIENT_ID="{myID}", maybe that could be more clear?

**Reply by unfor19** [reply]
- @lustoerk   I waited for someone to ask this question 😄  When it comes to this self-paced course, we don't really use any command that involves HTTPS from the host to minikube. All we did was using `kubectl` from WSL2/macOS. 
 
So why have I added it? Picture this - You want to access the Kubernetes APIs, from the Host with HTTPS, how would you do that? See [Access Clusters Using the Kubernetes API](https://kubernetes.io/docs/tasks/administer-cluster/access-cluster-api/). All the examples use `HTTP`, while with our setup we can use `HTTPS`, hooray for us :tada:

- You mean this?

   ```
   # Values from Google's Developer Console - the space at the beginning of the command is on purpose to keep it out from Bash's history
   OAUTH2_PROXY_CLIENT_ID="google_oauth2_project_client_id"
   OAUTH2_PROXY_CLIENT_SECRET="google_oauth2_project_client_secret"
   ```

**Reply by lustoerk** [reply]
Yeah, that was probably just really stupid :P i just thought "google_oauth2" has to be kept, "project" be replaced with the project name and "client_id" gets the value. But that failed and i realized the whole thing is the "client_id".
